util/settings/settings.go

```diff
@@ -408,6 +412,7 @@ func updateSettingsFromConfigMap(settings *ArgoCDSettings, argoCDCM *apiv1.Confi
 	settings.DexConfig = argoCDCM.Data[settingDexConfigKey]
 	settings.OIDCConfigRAW = argoCDCM.Data[settingsOIDCConfigKey]
 	settings.URL = argoCDCM.Data[settingURLKey]
+	settings.KustomizeBuildOptions = argoCDCM.Data[kustomizeBuildOptions]
```

Comment:
We are moving away from updating settings as a single call, in favor of individual accessor methods. I think we don't need this.

Reply:
If I were to remove this, where would I update the build option setting?

Reply:
Build options are manually set by editing configmap. We just need a new GetKustomizeBuildOptions() method to get the value of build options instead of the caller calling GetSettings().

Reply:
Ok, removed the line. I had gotten confused because I didn't realize `argoCDCM.Data[kustomizeBuildOptions]` pulled directly from the ConfigMap.